doc/source/user_guide/data_types.rst

```diff
@@ -73,6 +73,14 @@ issued::
    float64 to uint8
    array([  0, 128, 255], dtype=uint8)
 
+Warnings can be locally ignored with a context manager::
```

Comment:
My bad! I was looking at "at users:" in front of the table, not the stuff in front of code!